icons/flags/README.txt

```diff
@@ -0,0 +1,13 @@
+These flag icons were extracted from the Emoji One
+Git repository, found at
+
+    https://github.com/Ranks/emojione
+
+at commit
+
+    c0cbf73db93c87ac85a96c58905f88b4e342bb6f
+
+using the script found in the Mumble source tree
+found at
+
+    scripts/extract-emojione-flags.py
```

Comment:
No, but GitHub's display of the patch is ... not very friendly.

Maybe use https://github.com/mumble-voip/mumble/pull/1886.patch instead?